### PR TITLE
Added Integ test for password max length

### PR DESCRIPTION
**Pull request by benjiyue**
Addressed to #545

### CODE REVIEW

File: src/test/java/ru/mystamps/web/tests/cases/WhenAnonymousUserActivateAccount.java [hunk 1]
```diff
@@ -221,6 +222,15 @@ public void passwordShouldNotBeTooShort() {
 			.field("password")
 			.hasError(tr("value.too-short", PASSWORD_MIN_LENGTH));
 	}
+
+	@Test(groups = "invalid", dependsOnGroups = "std")
+	public void passwordShouldNotBeTooLong() {
+		page.activateAccount(null, null, StringUtils.repeat("0", PASSWORD_MAX_LENGTH + 1), null, null);
```
Comment (php-coder) [inline src/test/java/ru/mystamps/web/tests/cases/WhenAnonymousUserActivateAccount.java]: CheckStyle complains on this line:
>[WARNING] src/test/java/ru/mystamps/web/tests/cases/WhenAnonymousUserActivateAccount.java:[228] (sizes) LineLength: Line is longer than 100 characters (found 103).

**Comment by php-coder** [comment]
I see that you also created branch in **my** repository when you tried to revert your PR. I'm going to remove it.

**Comment by benjiyue** [comment]
I fixed the checkstyle issues

**Comment by php-coder** [comment]
Merged in f816b93b08810f789fff8e5af7d48c3ee12946f2 commit.

**Comment by php-coder** [comment]
Thank you for the pull request! Now you can clean your repo: https://github.com/php-coder/mystamps/wiki/after_merge_steps#remove-merged-branch

What is the next task that you want to work to?

**Pull request closed without merging**